shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/segment/index/IndexAssert.java

```diff
@@ -34,7 +37,7 @@
 public final class IndexAssert {
     
     /**
-     * Assert actual column segment is correct with expected column.
+     * Assert actual index segment is correct with expected index.
```

Comment:
Why modify this class

Reply:
Index can also have schema/owner (from my understanding, these two are identical), so add owner in IndexAssert.

![image](https://user-images.githubusercontent.com/66914151/123424086-a5fa1800-d5c0-11eb-8984-df1ec5ac6e6d.png)